network/policy/policy_windows.go

```diff
@@ -371,8 +373,15 @@ func GetHcnEndpointPolicies(policyType CNIPolicyType, policies []Policy, epInfoD
 				return hcnEndPointPolicies, err
 			}
 
-			hcnEndPointPolicies = append(hcnEndPointPolicies, endpointPolicy)
-			log.Printf("Successfully set the policy: %+v", endpointPolicy)
+			if isOutboundNatPolicy {
+				if snatAndSerialize || !enableMultiTenancy {
+					hcnEndPointPolicies = append(hcnEndPointPolicies, endpointPolicy)
```

Comment:
you can avoid duplicate code lines with something like
if outbound &&
(enableMultiTenancy && !snatAndSerialize) {
    continue
} else {
    hcnEndPointPolicies = append(hcnEndPointPolicies, endpointPolicy)
    log.Printf("Successfully set the policy: %+v", endpointPolicy)
}

duplicate code:
hcnEndPointPolicies = append(hcnEndPointPolicies, endpointPolicy)
log.Printf("Successfully set outbound NAT policy: %+v", endpointPolicy)